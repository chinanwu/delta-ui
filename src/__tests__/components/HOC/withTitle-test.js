// import { shallow } from 'enzyme';
import React from 'react';

// import { withTitle } from '../../../components/HOC/withTitle.jsx';

jest.unmock('../../../components/HOC/withTitle.jsx');

describe('withTitle HOC', () => {
	describe('behaviour', () => {
		xit('does its thing', () => {});
	});
});

//https://medium.com/swlh/components-testing-in-react-what-and-how-to-test-with-jest-and-enzyme-7c1cace99de5
