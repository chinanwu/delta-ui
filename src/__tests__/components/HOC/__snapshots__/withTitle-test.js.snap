// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withTitle HOC behaviour does its thing 1`] = `undefined`;

exports[`withTitle HOC rendering renders withTitle 1`] = `<withTitle />`;
